scripts/plot_docking_results.py

```diff
@@ -0,0 +1,75 @@
+import sys, os, argparse
+import numpy as np
+import plotly.express as px
+
+sys.path.append(
+    f"{os.path.dirname(os.path.dirname(os.path.abspath(__file__)))}"
+)
```

Comment:
We shouldn't need to do this, what are you trying to accomplish with this?

Reply:
added issue here: https://github.com/choderalab/covid-moonshot-ml/issues/48